tests/D2L.CodeStyle.TestAnalyzers.Tests/packages.config

```diff
@@ -1,13 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.CodeAnalysis.Analyzers" version="1.1.0" targetFramework="net461" />
+  <package id="Microsoft.CodeAnalysis.Analyzers" version="1.0.0" targetFramework="net461" />

```

Comment:
When I updated Nunit version to 2.6.4 it comes out a error to ask me to downgrade the version, but it should work with `1.1.0`, will change it back to `1.1.0`